pkg/kubelet/apis/stats/v1alpha1/types.go

```diff
@@ -198,6 +201,8 @@ type VolumeStats struct {
 	// Reference to the PVC, if one exists
 	// +optional
 	PVCRef *PVCReference `json:"pvcRef,omitempty"`
+	// FsType is the type of the Volume
+	FsType string
```

Comment:
Why not make this `FSType` instead of `string`?
Also we should probably be consistent with naming, either use `FSType` everywhere or `FsType` everywhere (I lean towards `FsType`).

Reply:
fixed